DashAI/back/models/hugging_face/distilbert_transformer.py

```diff
@@ -0,0 +1,176 @@
+import json
+import shutil
+
+import numpy as np
+from datasets import DatasetDict
+from sklearn.exceptions import NotFittedError
+from transformers import (
+    DistilBertForSequenceClassification,
+    DistilBertTokenizer,
+    Trainer,
+    TrainingArguments,
+)
+
+from DashAI.back.models.base_model import BaseModel
+from DashAI.back.models.text_classification_model import TextClassificationModel
+
+
+class DistilBertTransformer(BaseModel, TextClassificationModel):
+    """
+    Pre-trained transformer DistilBERT allowing English text classification
+    """
+
+    @classmethod
+    def get_schema(cls):
+        with open("DashAI/back/models/parameters/models_schemas/DistilBERT.json") as f:
+            cls.SCHEMA = json.load(f)
+        return cls.SCHEMA
```

Comment:
It is not necessary to return the schema once it is assigned into the class attributes.
```suggestion
```

Reply:
About this method I think it's a bit counter-intuitive to have a method called get and it doesn't deliver anything, we could talk about it. I think it would be better to just have that attribute without the need to call get.

Reply:
Ok, let's do it in a later pr.